src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -17,7 +16,53 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Constants / Defaults
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;
+;;; NOTE: We are making a decisive move away from overriding Jetty's
+;;; implicit default values for settings when downstream TK apps do not
+;;; explicitly provide values for them.  Please see the comments/tests in
+;;; `jetty9_default_config_test.clj` for full details.
+;;;
+;;; Below we are making a handful of deliberate exceptions to this rule,
+;;; but please do not perpetuate this pattern without a compelling reason to
+;;; do so.
+
+
+;;;
+;;; Host/port settings
+;;;
+;;; These are really common and fairly benign, and removing them would probably
+;;; only serve to make the bare configuration more onerous.
+;;;
+(def default-http-port 8080)
+(def default-https-port 8081)
+(def default-host "localhost")
```

Comment:
Yeah; and `localhost` is what we've always used here, didn't see a great reason to change it.